tsconfig.base.json

```diff
@@ -8,6 +8,7 @@
     "module": "ESNext",
     "declaration": true,
     "declarationMap": true,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "sourceMap": true,
```

Comment:
It triggers me that this is `.json` but actually supports trailing commas 🤯

Reply:
😨